.docker/cypress/spec/cypress/support/commands.js

```diff
@@ -65,6 +65,42 @@ Cypress.Commands.add("getCSRFToken", () => {
   })
 })
 
+Cypress.Commands.add('getInputByText', { prevSubject: 'optional'}, (subject, text, checkbox) => {
```

Comment:
Finde den Command so relativ verwirrend bzw. verstehe nicht auf den ersten Blick, was er macht.
Mein erster Vorschlag: zwei Commands, mit sprechenderen Namen: `nearestCheckbox` und `nearestInput`.
Ein Indiz für komplexen Code, den man in zwei Funktionen packen sollte/könnte ist jeweils ein if/else…

Zweiter Vorschlag (das wäre in jedem fall sinnvoll): Schreib doch einen kurzen Doc-Kommentar, der den Command beschreibt…

Reply:
Du hast absolut recht, der ganze Command ist überhaupt keine Augenweide. Ich mache zwei draus und dokumentiere noch.